api_views/test_func.py

```diff
@@ -1,8 +1,8 @@
 import requests
 
 
-def mock_bad_implementation(r: str, p: int, t: str):
-    a = f"https://api.github.com/repos/{r}/pulls/{p}"
-    h = {"Authorization": "Bearer " + t}
+def mock_bad_implementation(r: str, p: int, g: str):
+    a = f"https://api123.github.com/repos/{r}/pulls/{p}"
+    h = {"Authorization": "Bearer " + g}
```

Comment:
Sensitive information (Bearer token) should not be directly concatenated in string. Consider using a more secure method for handling authentication